python/ray/_private/ray_option_utils.py

```diff
@@ -188,7 +188,7 @@ def issubclass_safe(obj: Any, cls_: type) -> bool:
         if (x is None or x == "dynamic" or x == "streaming" or x >= 0)
         else "The keyword 'num_returns' only accepts None, a non-negative integer, or "
         '"dynamic" (for generators)',
```

Comment:
I think we want to hide "streaming". It will work without streaming if it is a generator. I will modify the comment when I hard deprecate the dynamic generator.